File: __tests__/fixtures/java/AllUsedImports.java
package com.example;

import java.util.List;
import java.util.ArrayList;
import java.util.Map;
import java.util.HashMap;

public class AllUsedImports {
    private List<String> items = new ArrayList<>();
    private Map<String, String> config = new HashMap<>();

    public void configure(String key, String value) {
        config.put(key, value);
    }
}

File: __tests__/fixtures/kotlin/AllUsedImports.kt
package com.example

import java.util.List
import java.util.ArrayList
import java.util.Map
import java.util.HashMap

class AllUsedImports {
    private val items: MutableList<String> = ArrayList()
    private val config: MutableMap<String, String> = HashMap()

    fun configure(key: String, value: String) {
        config[key] = value
    }
}
